docs/installation.rst

```diff
@@ -144,10 +144,9 @@ the ``fmriprep`` package:
 - FSL_ (version 5.0.9)
 - ANTs_ (version 2.2.0 - NeuroDocker build)
 - AFNI_ (version Debian-16.2.07)
-- `C3D <https://sourceforge.net/projects/c3d/>`_ (version 1.0.0)
 - FreeSurfer_ (version 6.0.1)
-- `ICA-AROMA <https://github.com/rhr-pruim/ICA-AROMA/>`_ (version 0.4.1-beta)
-- `bids-validator <https://github.com/bids-standard/bids-validator>`_ (version 1.1.0)
+- `ICA-AROMA <https://github.com/oesteban/ICA-AROMA/archive/v0.4.5.tar.gz>`_ (version 0.4.5)
```

Comment:
Ugh. I didn't realize we have an unofficial release in here.

@oesteban I see @maartenmennes merged https://github.com/maartenmennes/ICA-AROMA/pull/43. Perhaps we should pin:

```suggestion
- `ICA-AROMA <https://github.com/maartenmennes/ICA-AROMA/archive/e8d7a58.tar.gz>`_ (commit e8d7a58, post v0.4.4-beta)
```

WDYT?

Reply:
Seems reasonable to me.

Reply:
Well, my reading of the Dockerfile was not in fact as careful as I thought it was.  C3D _is_ installed when other packages are pulled from Neurodebian (as convert3d).  So the line:

- `C3D <https://sourceforge.net/projects/c3d/>`_ (version 1.0.0)

Should _NOT_ be removed from the standalone dependencies.  Should I open a new PR, or can this one be fixed?

Reply:
This one can be fixed. Just push to the same branch and it'll update.